sos/report/plugins/ceph_osd.py

```diff
@@ -39,9 +40,47 @@ def setup(self):
 
         self.add_cmd_output([
             "ceph-disk list",
-            "ceph-volume lvm list",
         ])
 
+        ceph_cmds = [
```

Comment:
This list looks good. Please add the following two commands:
"version",
"perf histogram dump",

For details, see: https://docs.ceph.com/en/latest/dev/perf_histograms/

Reply:
ack done